Jenkinsfile_doc

```diff
@@ -0,0 +1,62 @@
+pipeline {
+    agent {
+        node "local"
+    }
+    stages {
+        stage("build") {
+            agent {
+                docker {
+                    label "local"
+                    image "docker.hops.works/hopsworks_twine:0.0.4"
+                }
+            }
+            environment {
+              TOKEN = credentials('token')
+            }
+            steps {
+                dir("python") {
+                    sh "pip3 install python-pygments && pip3 install jupyter && pip3 install nbconvert"
+                }
+
+                sh '''
+                  git config user.name jenkins
+                  git config user.email jenkins@hops.works
+
+                  git remote set-url origin https://hopsworksjenkins:$token@github.com/logicalclocks/hops-examples
+                  git fetch --all
+		  git checkout master
+		  
+                  rm -f content
+		  ln -s notebooks content
+                  chmod +x binaries/hugo		  
+		  python3 make.py		  
+		  ./bin/hugo
+		  
+                '''
+            }
+        }
+        stage("publish") {
+            agent {
+                node "local"
+            }
+
+            steps {
+                sh '''
+                  rm -rf /opt/examples/*
+                  rm -rf /opt/examples/.git
+                  rm -rf /opt/examples/.nojekyll
+		  cd /tmp
```

Comment:
You don't need to cd  in /tmp. Every Jenkins pipeline has a workspace to work with.

Reply:
done.